index.html

```diff
@@ -0,0 +1,212 @@
+                  <li><a href="html/About.html">About</a></li>
+                  <li><a href="#">Contribution </a></li>
+                  <li><a href="#">Help</a></li>
+                  <li><a href="#">Ideas</a></li>
+                  <li><a href="#">News</a></li>
+                  <li><a href="#">CC Campain</a></li>
+                </ul>
+              </nav>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+    <!--Navbar coding side-->
+    <section class="closingTaging">
+      <div class="navbar">
+        <a href="#"><img src="svg/logo.svg" class="LogoChanga" alt="responsive-page-logo"></a>
+        <div class="nav-links">
+         <ul>
+           <img src="svg/IconCancel.svg" class="closeHumberger" alt="cloding-tag-icon" />
+           <li class="ColoringSide"><a class="Golinks" href="html/About.html">About</a></li>
+           <li ><a class="Golinks" href="#Portfolio">Contribution</a></li>
+           <li ><a class="Golinks" href="#Contact">Help</a></li>
+           <li class="ColoringSide"><a class="Golinks" href="#About">Ideas</a></li>
+           <li class="ColoringSide"><a class="Golinks" href="#About">News</a></li>
+         </ul>
+       </div>
+       <img src="svg/menuOpensvg.svg" class="menu-humberger" alt="opening-tag-icon" />
+    </div>
+    </section>
+       
+    <!--Here I'm implement main area that will  contain , and I'll  use grid system to play position-->
+    <main>
+      <div class="gridContainer">
+        <div class="firstGrid">
+          <h2 class="hDTagText">"Hi ! World Health"</h2>
+          <h1 class="textTag">WOLRD HEALTH COMMUNITY OF THE GLOBAL 2023</h1>
+        </div>
+        <div class="secondGrid">
+          <p>
+            This grant supports Boston-based community health centers’ efforts
+            to re-engage community members and patients into the health care
+            system who were affected by delays in care and temporary shutdowns
+            in services and are now experiencing negative health outcomes as a
+            result. Examples of interventions supported by the grant include
+            community health workers, patient navigators, and programming to
+            improve access to COVID-19 vaccination, testing, and treatment.
+          </p>
+        </div>
+        <div class="thirdGrid">
+          <h2>2023.11.18(THU)~19(FRI)</h2>
+          <p>@ National Team of world Health, ONU USA and more</p>
+        </div>
+        <div class="newtons-cradle">
+          <div class="newtons-cradle__dot"></div>
+          <div class="newtons-cradle__dot"></div>
+          <div class="newtons-cradle__dot"></div>
+          <div class="newtons-cradle__dot"></div>
+        </div>
+      </div>
+    </main>
+    <!--Here I'm implement side deatils area that will  contain , and I'll  use grid system to play position-->
+    <section class="firstSection">
+        <div class="Giveparaph">
+           <h1>Program Management</h1>
```

Comment:
- [OPTIONAL] To follow best semantics and conventions, there must be only one `h1` tag for the main title on each webpage. It also helps in SEO. 
I can see that you are taking multiple `h1`s in index and about pages. kindly consider to change all `h1` tags in both pages to `h2 to h6`, except the `h1` for the main title.